src/hooks/use-logs.ts

```diff
@@ -11,9 +13,10 @@ import { DEFAULT_LIST, ILog, logConverter } from '../data/data-types';
 import { checkedLogPath } from "../data/paths";
 import { saveMruListAndDeleteDraft } from "./use-account";
 
-export function useLogs(fBaseContext: IFirebaseContext, listName: string = DEFAULT_LIST) {
+export function useLogs(fBaseContext: IFirebaseContext, listName: string | undefined) {
   const logsCollection = collection(fBaseContext.db, checkedLogPath(fBaseContext)).withConverter(logConverter);
-  const [logsSnapshot, loading, error] = useCollectionData(logsCollection); //, where("list", "==", listName));
+  const logsQuery = listName ? query(logsCollection, where("list", "==", listName)) : logsCollection;
```

Comment:
Filtering on the query side is leading to a flash on UI re-render and also making it impossible to show a full dropdown of the lists after filtering occurs. Client-side list filtering is probably preferable. I could filter to a range of dates to reduce server calls, instead.